snippets/oadp-virtualization-supported-backup-types.adoc

```diff
@@ -0,0 +1,22 @@
+// Text snippet included in the following assemblies:
+//
+// ../backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[NOTE]
+====
+(oadp-full} with {VirtProductName} backup and restore support is limited to:
+
+* Container Storage Interface (CSI) backups
+
+* Container Storage Interface (CSI) backups with DataMover
+
+This support excludes:
+
+* File System backup and restore.
+
+* Volume Snapshot Backups and Restores.
+
+For more information, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc#oadp-backing-up-applications-restic-doc[Backing up applications with File System Backup: Kopia or Restic].
```

Comment:
text snippets cannot contain xrefs:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets

I would suggest making this an .Additional resources section: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences